skyportal/models/mmadetector.py

```diff
@@ -74,6 +78,186 @@ class MMADetector(Base):
         doc="GcnEvents associated with this detector.",
     )
 
+    spectra = relationship(
+        'MMADetectorSpectrum',
+        back_populates='detector',
+        cascade='save-update, merge, refresh-expire, expunge, delete',
+        single_parent=True,
+        passive_deletes=True,
+        order_by="MMADetectorSpectrum.start_time",
+        doc="MMADetectorSpectra of the object.",
+    )
+
 
 GcnEventMMADetector = join_model("gcnevents_mmadetectors", GcnEvent, MMADetector)
 GcnEventMMADetector.__doc__ = "Join table mapping GcnEvents to MMADetectors."
+
+
+class MMADetectorSpectrum(Base):
+    """Wavelength-dependent measurement of the sensitivity the detector."""
```

Comment:
I would argue that Frequency dependent is a much better way to look at it.

Reply:
OOps.